src/App/Pages/Vault/AddEditPageViewModel.cs

```diff
@@ -835,27 +836,44 @@ private void TriggerCipherChanged()
             TriggerPropertyChanged(nameof(Cipher), _additionalCipherProperties);
         }
 
-        private async void CheckPasswordAsync()
+        private async Task CheckPasswordAsync()
         {
-            if (!(Page as BaseContentPage).DoOnce())
-            {
-                return;
-            }
             if (string.IsNullOrWhiteSpace(Cipher.Login?.Password))
             {
                 return;
             }
+
             await _deviceActionService.ShowLoadingAsync(AppResources.CheckingPassword);
-            var matches = await _auditService.PasswordLeakedAsync(Cipher.Login.Password);
-            await _deviceActionService.HideLoadingAsync();
-            if (matches > 0)
+            try
```

Comment:
the try should begin at the beginning of the method, otherwise if an exception is raised in let's say `_deviceActionService.ShowLoadingAsync` it won't be caught by the catch